pages/standards.tsx

```diff
@@ -111,6 +113,16 @@ const Standards = (data: StandardsProps) => {
                                         Imaging
                                     </Link>
                                 </li>
+                                <li>
+                                    <Link href="https://sage-bionetworks.github.io/schema_visualization/">
+                                        <a>
```

Comment:
Should we force opening the schema visualizer on a new tab? We may need to do something like this for that: https://stackoverflow.com/a/71029662

Reply:
Yeah that makes sense. Thanks @onursumer ! I just made the change.